### PR TITLE
Insert UDF with accuracy

### PR DESCRIPTION
**Pull request by NetraGhaisas**

### CODE REVIEW

File: eva/optimizer/operators.py [hunk 1]
```diff
@@ -1043,3 +1054,78 @@ def __eq__(self, other):
 
     def __hash__(self) -> int:
         return hash((super().__hash__(), self.show_type))
+
```
Comment (aravindhank11) [inline eva/optimizer/operators.py]: Is this still relevant?

Comment (aravindhank11) [reply eva/optimizer/operators.py]: If not, can we please remove?

File: eva/optimizer/operators.py [hunk 2]
```diff
@@ -54,6 +54,7 @@ class OperatorType(IntEnum):
     LOGICAL_SHOW = auto()
     LOGICALDROPUDF = auto()
     LOGICALDELIMITER = auto()
+    LOGICAL_CREATE_UDF_PROFILER_SAMPLE = auto()
```
Comment (aravindhank11) [inline eva/optimizer/operators.py]: Not needed?

File: eva/udfs/udf_bootstrap_queries.py [hunk 3]
```diff
@@ -23,6 +23,7 @@
                   INPUT  (Frame_Array NDARRAY INT8(3, ANYDIM, ANYDIM))
                   OUTPUT (label NDARRAY STR(1))
                   TYPE  Classification
+                  ACCURACY 0
```
Comment (aravindhank11) [inline eva/udfs/udf_bootstrap_queries.py:26]: 0??? haha

**Review by aravindhank11 — APPROVED** [review]
Awesome! Thanks for the changes, @NetraGhaisas. Except for a few dead code (I believe) rest all LGTM. 

Can you please review & fix those comments prior to proceeding to merge?

File: eva/optimizer/operators.py (hunk 1 above)
Comment (NetraGhaisas) [reply eva/optimizer/operators.py]: Right, not relevant. Removed this code

**Comment by NetraGhaisas** [comment]
Thanks for reviewing @aravindhank11! Removed dead code, everything should be up to date now.

**Review by aravindhank11 — APPROVED** [review]

**Pull request merged**